introduction.md

```diff
@@ -0,0 +1,7 @@
+Hello! My name is Gaby. I am doing this Github Tutorial where I have to give an introudction about myself. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.